todo/src/actions/index.js

```diff
@@ -0,0 +1,20 @@
+export const addTodo = todoItem => {
+  return {
+    type: "ADD_TODO",
+    payload: {
+      task: todoItem,
```

Comment:
after going through so much effort to generate a new ID I am surprised to see that it's not even being passed to the redux store ;(